vsts/sdl.yaml

```diff
@@ -13,6 +13,12 @@ jobs:
     vmImage: windows-latest
     timeoutInMinutes: 60
   steps:
+  - task: Maven@3
```

Comment:
When we bump versions the SDL task will fail until we release the packages to Maven. This will ensure that we're always building the latest packages for SDL to scan.